modules/labs-react/common/README.md

```diff
@@ -16,3 +16,41 @@ yarn add @workday/canvas-kit-labs-react/common
 
 For more detailed information on this component, please refer to the
 [storybook documentation](https://workday.github.io/canvas-kit/?path=/docs/labs-common)
+
+## useThemeRTL
+
+This is a way to automatically transform CSS, applied using either Emotion's css attribute or by a
+style attribute, when there is a RTL theme in use. Instead of authoring two sets of CSS objects, one
+for each language direction. Now you can author the LTR version and this hook will automatically
+create the RTL counterpart for you on the fly. This works for both object and string template
+Emotion methods.
+
+For convenience we also export a ComponentStyles type if you want to organize your styles all
+together.
+
+### Usage
+
+```tsx
+/** @jsx jsx */
+import {jsx} from '@emotion/core';
+import {ComponentStyles, useThemeRTL} from '@workday/canvas-kit-labs-react/common';
+const styles: ComponentStyles = {
+  panel: {
+    margin: '8px',
+  },
+  top: {
+    top: '8px',
+  },
+  red: {
+    color: 'red',
+  },
```

Comment:
import {space} from '@workday/canvas-kit-react/tokens';

```suggestion
const styles: ComponentStyles = {
  panel: {
     // `themeRTL` will transform this to `marginRight` when the direction is `RTL`
    marginLeft: space.xxs,
  },
  top: {
    top: space.xxs,
  },
  red: {
    color: 'red',
  },
```

This is a good, simple example, but I'd like to make a couple slight alterations:

* use space tokens just to encourage the idea of using token values
* switch `margin` to `marginLeft` as that attr will be updated for RTL